plugins/linux-v4l2/v4l2-output.c

```diff
@@ -199,7 +199,12 @@ static bool virtualcam_start(void *data)
 	for (int i = 0; i < n; i++) {
 		char device[32];
 
-		snprintf(device, 32, "/dev/%s", list[i]->d_name);
+		// Use the return value of snprintf to prevent truncation warning
+		int written = snprintf(device, 32, "/dev/%s", list[i]->d_name);
+		if (written > 32)
+			blog(LOG_DEBUG,
+			     "v4l2-output: A format truncation may have happened,"
+			     " this can be ignored since it is quite improbable");
```

Comment:
Use:
"A format truncation may have occurred."
" This can be ignored since it is quite improbable."

Use this here and in the other location in obs-ffmpeg-vaapi.c.

Reply:
Thank you, changes done.